File: calculator/src/test/java/com/grijesh/calculator/test/BDDRunner.java
package com.grijesh.calculator.test;

import cucumber.api.CucumberOptions;
import cucumber.api.junit.Cucumber;
import org.junit.runner.RunWith;

/**
 * Created by grijesh on 17/10/15.
 */

@RunWith(Cucumber.class)
@CucumberOptions(
        monochrome = true,
        features = "src/test/resources/features/",
        plugin = {"pretty", "html:build/results"},
        dryRun = false,
        glue = "com.grijesh.calculator.test.calculator"
)
public class BDDRunner {
}

File: scala-calculator/src/test/scala/com/grijesh/calculator/test/BddRunner.scala
package com.grijesh.calculator.test

/**
  * Created by grijesh on 13/3/16.
  */

import cucumber.api.CucumberOptions
import org.junit.runner.RunWith
import cucumber.api.junit.Cucumber

@RunWith(classOf[Cucumber])
@CucumberOptions(
  monochrome = true,
  features = Array("src/test/resources/features/"),
  plugin = Array("pretty", "html:build/results"),
  dryRun = false,
  glue = Array("com.grijesh.calculator.test")
)
class BddRunner
